iothub_client/inc/iothub_client.h

```diff
@@ -37,354 +45,109 @@ extern "C"
 #endif
 
     /**
-    * @brief    Creates a IoT Hub client for communication with an existing
-    *             IoT Hub using the specified connection string parameter.
-    *
-    * @param    connectionString    Pointer to a character string
-    * @param    protocol            Function pointer for protocol implementation
-    *
-    *            Sample connection string:
-    *                <blockquote>
-    *                    <pre>HostName=[IoT Hub name goes here].[IoT Hub suffix goes here, e.g., private.azure-devices-int.net];DeviceId=[Device ID goes here];SharedAccessKey=[Device key goes here];</pre>
-    *                   <pre>HostName=[IoT Hub name goes here].[IoT Hub suffix goes here, e.g., private.azure-devices-int.net];DeviceId=[Device ID goes here];SharedAccessSignature=SharedAccessSignature sr=[IoT Hub name goes here].[IoT Hub suffix goes here, e.g., private.azure-devices-int.net]/devices/[Device ID goes here]&sig=[SAS Token goes here]&se=[Expiry Time goes here];</pre>
-    *                </blockquote>
-    *
-    * @return    A non-NULL @c IOTHUB_CLIENT_HANDLE value that is used when
-    *             invoking other functions for IoT Hub client and @c NULL on failure.
+    * @warning IoTHubClient_CreateFromConnectionString is deprecated.  Use IoTHubDeviceClient_CreateFromConnectionString instead.
```

Comment:
Additional option is the `@deprecated` tag in doxygen.

![image](https://user-images.githubusercontent.com/36710865/97062150-76d16980-154e-11eb-93b8-e05fe76e1a0c.png)

Reply:
I really like this.  Changing.